src/AppShop.java

```diff
@@ -0,0 +1,31 @@
+import java.util.List;
+
+public class AppShop {
+    public static void main(String[] args) {
+        List<Product> productsInShop = StockOfProducts.getProducts();
+
+        System.out.println("Find books to current sale price: ");
+        Controller.getBooksToCurrentSalePrise(productsInShop);
+        System.out.println("-----------------------------------");
```

Comment:
you can save "---------------------" to the var and reuse it instead of type it over nad over again